Gruntfile.js

```diff
@@ -195,37 +202,30 @@ module.exports = function( grunt ) {
 	} );
 
 	grunt.registerTask( "lint", [
-		"jsonlint",
-		runIfNewNode( "eslint:dev" ),
-		runIfNewNode( "eslint:dist" )
+		"newer:jsonlint",
+		runIfNewNode( "newer:eslint:dev" ),
+		runIfNewNode( "newer:eslint:dist" )
 	] );
 
-	grunt.registerTask( "test_fast", [ runIfNewNode( "node_smoke_tests" ) ] );
```

Comment:
> I don't think we should land PRs not LGTM'd by at least 2 core members after only a day of submitting a PR

We don't have this policy, it would be cool to have something like though it would be good if you put this on the next team meeting agenda. 

`test_fast` task will be putted back, since it is questionable change, this is why I asked for second opinion. These changes are pretty trivial and don't affect the source so I don't see the reason, why we want pull requests to be in the queue for couple of weeks